keyboards/crkbd/serial.c

```diff
@@ -0,0 +1,238 @@
+/*
+ * WARNING: be careful changing this code, it is very timing dependent
+ */
+
+#ifndef F_CPU
+#define F_CPU 16000000
+#endif
+
+#include <avr/io.h>
+#include <avr/interrupt.h>
+#include <util/delay.h>
+#include <stdbool.h>
+#include "serial.h"
+
+#ifdef USE_SERIAL
```

Comment:
This should be `#ifndef USE_I2C`

Reply:
This file includes logic code for serial only.
Why cheking the flag of I2C here?